mlir/include/mlir/Dialect/AMDGPU/Utils/Chipset.h

```diff
@@ -49,6 +49,14 @@ struct Chipset {
 #undef DEFINE_COMP_OPERATOR
 };
 
+inline bool isGfx940Series(const Chipset &chipset) {
+  return chipset.majorVersion == 9 && chipset.minorVersion == 4;
+}
+inline bool hasOcpFp8(const Chipset &chipset) {
```

Comment:
Let's make these methods on `Chipset` and not freestanding functions that'll pollute the namespace.

Reply:
This does contradict a previous comment by @kuhar
https://github.com/llvm/llvm-project/pull/106160#discussion_r1763442491

Reply:
Will then we'll keep `hasOcpFp8` as requested, then. No big deal.